Loop/View Controllers/BolusViewController.swift

```diff
@@ -51,6 +53,13 @@ final class BolusViewController: ChartsTableViewController, IdentifiableClass, U
 
                     self?.reloadData(animated: true)
                 }
+            },
+            NotificationCenter.default.addObserver(forName: .HKUserPreferencesDidChange, object: deviceManager.loopManager.glucoseStore.healthStore, queue: nil) {[weak self] _ in
```

Comment:
Have the enclosing view controller keep track of whether the chart needs to be refreshed due to glucose unit changes